build_scripts/staging_build.sh

```diff
@@ -0,0 +1,18 @@
+echo "Checking out to master"
+git checkout master
+
+echo "Rebase with origin"
+git fetch origin
+
+echo "Switching to tag"
+git checkout -b tag_$1 $1
+
+echo "fetching data from gigalixir"
+git pull staging master
```

Comment:
make staging an argument. so we can use the same script for staging/gigalixir or whatever the person executing it has called those remotes